tests/examples/intersphinx/conf.py

```diff
@@ -6,10 +6,15 @@
     'sphinx.ext.intersphinx',
 ]
 
-hoverxref_intersphinx = True
+hoverxref_intersphinx = [
+    'python',
+]
```

Comment:
Wouldn't it be more natural to use `readthedocs` for this test instead, since `python` is not hosted on RtD at the moment?

Reply:
Maybe. It doesn't really affect the test here. It's just a test to check that the backend is generating the HTML tag as we expect. It doesn't really make the request or anything.

Reply:
_On the other hand_, the client should show a nicer error if the request returns 404 😄